src/pages/ReimbursementAccount/PersonalInfo/substeps/Address.tsx

```diff
@@ -60,10 +62,14 @@ function Address({reimbursementAccount, onNext, isEditing}: AddressProps) {
         zipCode: reimbursementAccount?.achData?.[personalInfoStepKey.ZIP_CODE] ?? '',
     };
 
-    const handleSubmit = (values: BankAccounts.PersonalAddress) => {
-        BankAccounts.addPersonalAddressForDraft(values);
-        onNext();
-    };
+    const handleSubmit = useReimbursementAccountStepFormSubmit({
```

Comment:
this should be a function

Reply:
actually, even though this is correct, I think following a more standard syntax of returning a callback from the hook and then assigning it to the `handleSubmit` or directly to `onSubmit` would be more readable

Reply:
Could You send an example? I am not sure what do You mean

Reply:
This hook returns a callback, which I've saved into variable 'handleSubmit' and I used this variable in prop.

Reply:
that's what I mean by saying that it's correct.

what I had in mind was something more along the lines of `useSubstep` hook that you would destructure a callback from the hook as in `const {callback} = useReimbursementAccountStepFormSubmit({...})`. The current syntax got me confused in the beginning. Again, this is purely syntax-wise to cater to more common pattern in my opinion, feel free to disagree. wdyt @MrMuzyk ?

Reply:
but hook returns only a callback, not a object.  I don't think that this callback will return something else.

Reply:
I understand that, however please have a look at other hooks in the project, even if they are returning one thing it follows this syntax `return { something }`

Reply:
still, hooks can return whatever you want.

Reply:
check useDebounce or useWaitForNavigation for examples. ALso, hooks often return an array, so convention that You mention doesn't exists

Reply:
they can, that's why I am saying the code is correct, what I am asking is to change the syntax so it's not confusing and follows the project's pattern for those

Reply:
It follows project patterns, please take a look on examples

Reply:
I feel like we should follow already existing and more popular pattern in the codebase

Reply:
I think if the purpose of the hook is creating only function there is no reason to return an object that must be destructured every time. Please take a look at the others hooks with function - they are returning only function

Reply:
again, I am not talking about hooks with function but hooks that return _**EVEN**_ one thing, be it a variable or a function, they mostly do follow the destructurisation pattern, as per examples I provided above

Reply:
I've sent examples with hooks that don't follow the approach mentioned by You. There is more in our codebase. 
Is there a rule in the contribution guide that clarifies this? If not - I think this rule doesn't exist in this project, so can't be a discussion point.

Can you highlight other benefits of Your proposition?

Reply:
So, if we don't have a convention in the guide and there is no performance-related reason, I want to keep current state. 
In the react comunnity in common approach hooks returns only needed things directly(e.g https://usehooks.com/) so I goto follow this one.